src/pages/index.js

```diff
@@ -3,20 +3,22 @@ import { Link } from "gatsby"
 import Layout from "../components/layout"
 import Image from "../components/image"
 import SEO from "../components/seo"
-import ExampleComponent from "../components/example/Example"
+import ExampleComponent from '../components/example/Example'
+import { Events } from "../components/Events"
 
 const IndexPage = () => (
-  <Layout>
-    <SEO title="Home" />
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <ExampleComponent componentTitle="Zombie Apocalypse" />
-    <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
-      <Image />
-    </div>
-    <Link to="/page-2/">Go to page 2</Link>
-  </Layout>
+  // <Layout>
+  //   <SEO title="Home" />
+  //   <h1>Hi people</h1>
+  //   <p>Welcome to your new Gatsby site.</p>
+  //   <p>Now go build something great.</p>
+  //   <ExampleComponent componentTitle="Zombie Apocalypse" />
+  //   <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
+  //     <Image />
+  //   </div>
+  <Events />
+  //   <Link to="/page-2/">Go to page 2</Link>
+  // </Layout>
```

Comment:
You could probably check the comment unless you're saving it for a particular reason. Half the time I do this I never use it ever again.

Reply:
Nope. Not really saving it for anything. Just did it to more easily visualize our components we were writing. I'll take it out with the styles as well before I check in with Ben to merge.